javascript-new/javascript-new-quiz-notes.md

```diff
@@ -0,0 +1,42 @@
+# javascript-constructors-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What does the `new` operator do?
+  It creates an instance of a user-defined object type or of one of the built-in object types that has a constructor function.
+
+- What property of JavaScript functions can store shared behavior for instances created with `new`?
+  this
```

Comment:
Not quite! Think about what property we can check out to find additional properties and methods. When I try calling a method on an object, and it doesn't exist on that object, which property is it going to check and keep going down through to try and find it?